.github/workflows/osml-imagery-toolkit-build.yml

```diff
@@ -1,4 +1,4 @@
-name: "build"
+name: OSML Imagery Toolkit Build Workflow
```

Comment:
I would revert this back to `build` otherwise it would be long in the readme as shown here:
- https://github.com/aws-solutions-library-samples/osml-imagery-toolkit/tree/main

Reply:
Done